library/ajax/drug_autocomplete/search.php

```diff
@@ -0,0 +1,45 @@
+<?php
```

Comment:
I removed the js folder that was in the drug_autocomplete. It was not needed it came with the sample code.